integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDBOperator.java

```diff
@@ -0,0 +1,740 @@
+  void  testFmwModelInImageWithDbOperator() {
+    // Create the repo secret to pull the image
+    // this secret is used only for non-kind cluster
+    createOcirRepoSecret(fmwDomainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(fmwAminSecretName,
+        fmwDomainNamespace,
+        "weblogic",
+        "welcome1"),
+        String.format("createSecret failed for %s", fmwAminSecretName));
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(fmwEncryptionSecretName,
+        fmwDomainNamespace,
+        "weblogicenc",
+        "weblogicenc"),
+        String.format("createSecret failed for %s", fmwEncryptionSecretName));
+
+    // create RCU access secret
+    logger.info("Creating RCU access secret: {0}, with prefix: {1}, dbUrl: {2}, schemapassword: {3})",
+        rcuaccessSecretName, RCUSCHEMAPREFIX, RCUSCHEMAPASSWORD, dbUrl);
+    assertDoesNotThrow(() -> createRcuAccessSecret(
+        rcuaccessSecretName,
+        fmwDomainNamespace,
+        RCUSCHEMAPREFIX,
+        RCUSCHEMAPASSWORD,
+        dbUrl),
+        String.format("createSecret failed for %s", rcuaccessSecretName));
+
+    logger.info("Create OPSS wallet password secret");
+    assertDoesNotThrow(() -> createOpsswalletpasswordSecret(
+        opsswalletpassSecretName,
+        fmwDomainNamespace,
+        "welcome1"),
+        String.format("createSecret failed for %s", opsswalletpassSecretName));
+
+    logger.info("Create an image with jrf model file");
+    final List<String> modelList = Collections.singletonList(MODEL_DIR + "/" + modelFile);
+    fmwMiiImage = createMiiImageAndVerify(
+        "jrf-mii-image",
+        modelList,
+        Collections.singletonList(MII_BASIC_APP_NAME),
+        FMWINFRA_IMAGE_NAME,
+        FMWINFRA_IMAGE_TAG,
+        "JRF",
+        false);
+
+    // push the image to a registry to make it accessible in multi-node cluster
+    dockerLoginAndPushImageToRegistry(fmwMiiImage);
+
+    // create the domain object
+    Domain domain = FmwUtils.createDomainResource(fmwDomainUid,
+        fmwDomainNamespace,
+        fmwAminSecretName,
+        OCIR_SECRET_NAME,
+        fmwEncryptionSecretName,
+        rcuaccessSecretName,
+        opsswalletpassSecretName,
+        replicaCount,
+        fmwMiiImage);
+
+    createDomainAndVerify(domain, fmwDomainNamespace);
+
+    //verify the introspector pod is created and runs
+    logger.info("Verifying introspector pod is created, runs");
+    String introspectPodNameBase = getIntrospectJobName(fmwDomainUid);
+    logger.info("Checking introspector pod exists and introspect version");
+    checkPodExists(introspectPodNameBase, fmwDomainUid, fmwDomainNamespace);
+    String introspectPodName = assertDoesNotThrow(() -> Kubernetes.listPods(fmwDomainNamespace, null)
```

Comment:
use static import listPods

Reply:
I think we have a common method to get the introspectPodName

Reply:
this part of the code is removed since we don't really care about introspector here